tests/test_api.py

```diff
@@ -229,3 +230,41 @@ def label(row):
         "950-NonConvex=2",
         "912-NoRelHeurWork=60",
     }
+
+
+def test_tee():
```

Comment:
Should the name for this test change? I am not sure what "tee" represents. Maybe "test_rewrite_filenames"?

It might also make sense to add a log with default parameters and no model name to the `/tests/assets/combined/` folder to ensure the files are named correctly in these cases too.

Reply:
> Should the name for this test change? I am not sure what "tee" represents. Maybe "test_rewrite_filenames"?

It was a lazy reference to unix `tee` since one input data stream goes out to two places :-) Fixed!

> It might also make sense to add a log with default parameters and no model name to the /tests/assets/combined/ folder to ensure the files are named correctly in these cases too.

Good thought, done. Test catches my stray 's' now.